ArduCopter/takeoff.cpp

```diff
@@ -218,6 +223,11 @@ void _AutoTakeoff::run()
     if (complete) {
         const Vector3p& _complete_pos = copter.pos_control->get_pos_target_cm();
         complete_pos = Vector3p{_complete_pos.x, _complete_pos.y, pos_z};
+#if AP_TIE_DOWN_CLAMPS_ENABLED
```

Comment:
@lthall why do we need this block?  I'm not sure how we get to here without going through the init function which should already have released the clamps.

Reply:
This sets the clamps back to the closed position after the take-off has completed. That way the state matches the switch position if it is being used and if it isn't the clamp is ready for reattachment when the aircraft lands again.

Reply:
I've added a comment, thanks.